tests/config/require-standalone.js

```diff
@@ -1,25 +1,51 @@
 "use strict";
 
-const prettier = require("prettier/standalone");
-const babelPlugin = require("prettier/parser-babel");
-const solidityPlugin = require("../../src/index");
+const path = require("path");
+const vm = require("vm");
+const createSandBox = require("./utils/create-sandbox");
 
+const sandbox = createSandBox({
+  files: [
+    path.join(path.dirname(require.resolve("prettier")), "standalone.js"),
+    path.join(path.dirname(require.resolve("prettier")), "parser-babel.js"),
+    path.join(__dirname, "../../dist/standalone.js"),
+  ],
+});
+
+// TODO: maybe expose (and write tests) for `format`, `utils`, and
```

Comment:
Why do we need to expose them for testability? Can’t we test the parent component as a whole?

Reply:
this comment has been copied from prettier's `tests/config/require-standalone.js` and changes are minimal